packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -763,188 +756,86 @@ export function TransferPanelMain({
-          return false
-        }
+    // we hide local networks, these are for dev only and should be accessed from the wallet
+    const chainIdsToHide = [ChainId.Local, ChainId.ArbitrumLocal, 1338]
 
-        if (isOrbitChain && isDestinationChainList && isSourceArbitrumNova) {
-          return false
-        }
+    function isChildChain(
+      chain: L2Network | ParentChain | undefined
+    ): chain is L2Network {
+      if (!chain) {
+        return false
+      }
+      return typeof (chain as L2Network).partnerChainID !== 'undefined'
+    }
 
-        // If this is the Source network list options
-        // and the selected source is an Arbitrum Base chain
-        // we don't show Orbit chains except for the current Destination Orbit chain on the same dropdown
-        if (
-          isSelectedArbitrumChain &&
-          isSourceChainList &&
-          isOrbitChain &&
-          !isSameAsDestinationChain
-        ) {
-          return false
-        }
+    const chains = getChains()
 
-        // If this is the Destination network list options
-        // and the selected destination is an Arbitrum chain
-        // we don't show Orbit chains except for the current Source Orbit chain on the same dropdown
-        if (
-          isSelectedArbitrumChain &&
-          isDestinationChainList &&
-          isOrbitChain &&
-          !isSameAsSourceChain
-        ) {
-          return false
-        }
-
-        // If the source chain is an Orbit Chain,
-        // and this is the Destination network list options
-        if (isSourceOrbitChain && isDestinationChainList) {
-          // we do not show Ethereum Mainnet or Testnet as options
-          if (isEthereumMainnetOrTestnet) {
+    function getSourceChains() {
+      return Object.keys(chains)
+        .filter(chainId => {
+          if (chainIdsToHide.includes(Number(chainId))) {
             return false
           }
-          // we do not show other Orbit chains as options
-          if (isOrbitChain && !isSameAsSourceChain) {
-            return false
+          // don't show testnet networks if testnet mode is off
+          if (!isTestnetMode) {
+            return !isNetwork(Number(chainId)).isTestnet
           }
-        }
+          return true
+        })
+        .map(chainId => getWagmiChain(Number(chainId)))
+    }
 
-        // If the destination chain is an Orbit Chain,
-        // and this is the Source network list options
-        if (isDestinationOrbitChain && isSourceChainList) {
-          // we do not show Ethereum Mainnet or Testnet as options
-          if (isEthereumMainnetOrTestnet) {
-            return false
-          }
-          // we do not show other Orbit chains as options
-          if (isOrbitChain && !isSameAsDestinationChain) {
-            return false
-          }
-        }
+    function getDestinationChains() {
+      const sourceChain = chains[networks.sourceChain.id]
```

Comment:
We don't take this from `useNetworksRelationship` because we need an SDK object that has `partnerChainID` on it.

In general I think we can kill `useNetworksRelationship` in favour of having SDK as the source of truth for that.